mx.py

```diff
@@ -15094,7 +15109,7 @@ def processDep(dep, edge):
             out.element('target', data='jar')
             out.element('clean-target', data='clean')
             out.element('id', data='jar')
-            out.close('reference')
+            out.close('reference') #pylint: disable=too-many-function-args
```

Comment:
False positive, thinks we're using a different type for `out` then we actually are, since it's defined in an enclosing scope